guides/source/action_text_overview.md

```diff
@@ -113,16 +113,8 @@ end
 
 ## Rendering Rich Text content
 
-Action Text will sanitize and render rich content on your behalf.
```

Comment:
@jonathanhefner @p8 with the removal of this line, the Action Text Overview only mentions "sanitization" in one place:

https://github.com/rails/rails/blob/00dfa109d6a5f4fb13d745f83d5cc779eba3b352/guides/source/action_text_overview.md#L95-L99

Surprisingly, neither [ActionText::RichText](https://github.com/rails/rails/blob/00dfa109d6a5f4fb13d745f83d5cc779eba3b352/actiontext/app/models/action_text/rich_text.rb) nor [ActionText::Content](https://github.com/rails/rails/blob/00dfa109d6a5f4fb13d745f83d5cc779eba3b352/actiontext/lib/action_text/content.rb) mention anything about sanitizing content in their RDoc comments. 

[Securing Rails Applications](https://edgeguides.rubyonrails.org/security.html) doesn't mention Action Text at all, nor does it mention that it's is serialized by [rails-html-sanitizer](https://github.com/rails/rails-html-sanitizer/).

Was the removal of this line intentional, or accidental? Where would be the most appropriate place to improve the Action Text documentation with regard to its security implications?

Reply:
I've opened https://github.com/rails/rails/pull/49696 to try and expand upon sanitization support.